frontend/src/views/omni/Overview/Overview.vue

```diff
@@ -109,11 +109,9 @@ included in the LICENSE file.
           <div class="flex-col space-y-6">
             <div class="overview-card p-6 flex flex-col gap-5 place-items-stretch" style="width: 350px">
               <div class="text-naturals-N14 text-sm">Add Machines</div>
-              <t-button icon="long-arrow-down" iconPosition="left" @click="() => openDownloadIso()">Download
-                Installation Media
-              </t-button>
-              <t-button icon="copy" iconPosition="left" @click="() => copyValue(items[0]?.spec?.args)">Copy Kernel
-                Parameters</t-button>
+              <t-button icon="long-arrow-down" iconPosition="left" @click="() => openDownloadIso()">Download Installation Media</t-button>
+              <t-button icon="long-arrow-down" iconPosition="left" @click="() => downloadMachineConfig(items[0]?.spec)">Download Machine Config</t-button>
```

Comment:
Maybe `Download Machine Join Config` will be more clear about what this config does.